lmdeploy/pytorch/models/utils/cudagraph.py

```diff
@@ -60,10 +60,17 @@ def make_buffers_cudagraph(self, graph_meta: CudaGraphMeta, *args, **kwargs) ->
         input_buffers: BuffType = dict()
         input_buffers['input_ids'] = torch.zeros(1, max_tokens, dtype=torch.int64, device=device)
         input_buffers['position_ids'] = torch.zeros((1, max_tokens), dtype=torch.int64, device=device)
+        if getattr(self.config, 'use_flash_mla', False) is True:
+            import flash_mla_cuda
 
-        input_buffers['block_offsets'] = torch.zeros((max_batches, num_blocks), dtype=torch.int64, device=device)
+            # create buffers for flash mla
+            input_buffers['tile_scheduler_metadata'], input_buffers['num_splits'] = flash_mla_cuda.get_mla_metadata(
```

Comment:
Do we have to call any kernel on creating buffers?

Reply:
We can set them to [num_sms, num_expers] and [batch_size+1]. But I am not sure if there is any risk.